src/Blankslate/Blankslate.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import {BookIcon} from '@primer/octicons-react'
+import {Meta, Story} from '@storybook/react'
+import React from 'react'
+import {Blankslate} from '../Blankslate'
+import {ComponentProps} from '../utils/types'
+
+export default {
+  title: 'Drafts/Components/Blankslate',
+  component: Blankslate,
+  subcomponents: {
+    'Blankslate.Visual': Blankslate.Visual,
+    'Blankslate.Heading': Blankslate.Heading,
+    'Blankslate.Description': Blankslate.Description,
+    'Blankslate.PrimaryAction': Blankslate.PrimaryAction,
+    'Blankslate.SecondaryAction': Blankslate.SecondaryAction,
+  },
+} as Meta<typeof Blankslate>
+
+export const Default: Story<ComponentProps<typeof Blankslate>> = args => (
```

Comment:
Default stories should accept args. Maybe we can create a Playground story that accepts args.